BACKEND/userAuth/src/test/java/com/apptime/auth/controller/TaskManagerTest.java

```diff
@@ -113,6 +127,48 @@ public void testGetTasks() throws Exception {
         verifyResultsWithName(actions, 0, taskWithDifferentUser.getName());
     }
 
+    @Test
+    public void testshowAddedSince() throws Exception {
+        FormatedDate start = new FormatedDate();
+        String sDate="2020-03-22";
+        SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
+        Date date=dateFormat.parse(sDate);
+        start.setDate(date);
+        Task task1 = createTaskWithDueDate(start,"task1");
+        String body = (new ObjectMapper()).valueToTree(start).toString();
+        mockMvc.perform(MockMvcRequestBuilders.post("/tasks/due/start")
+            .with(user(USERNAME))
+            .contentType(MediaType.APPLICATION_JSON)
+                .content(body)
```

Comment:
The request body is unnecessary for the API.

Reply:
You can't send the request without body. it errors out. try it.

Reply:
Oh sorry it is my mistake. I thought it was a different api. never mind.